SYCL/ESIMD/api/functional/ctors/ctor_fill_accuracy_fp_extra.cpp

```diff
@@ -0,0 +1,65 @@
+//==------- ctor_fill_accuracy_fp_extra.cpp  - DPC++ ESIMD on-device test --==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu, level_zero
+// XREQUIRES: gpu
+// TODO gpu and level_zero in REQUIRES due to only this platforms supported yet.
+// The current "REQUIRES" should be replaced with "gpu" only as mentioned in
+// "XREQUIRES".
+// UNSUPPORTED: cuda, hip
+// XRUN: %clangxx -fsycl %s -fsycl-device-code-split=per_kernel -o %t.out
+// XRUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: false
+// XFAIL: *
+// TODO The simd can't be constructed with sycl::half data type. The issue was
```

Comment:
Please try the newer compiler. sycl::half should be working fine now.

Reply:
This discuss related to all tests, so suggest discuss it in the one place, in this your comment: [discussion in PR#687](https://github.com/intel/llvm-test-suite/pull/687#discussion_r778299304)